[Components][ClassLoader] remove DebugClassLoader

xabbuh opened the pull request:
| Q | A |
| --- | --- |
| Doc fix? | no |
| New docs? | no |
| Applies to | 3.0+ |
| Fixed tickets |  |

The `DebugClassLoader` has been moved to the Debug component with
Symfony 2.4 and will be removed in 3.0 (see symfony/symfony#13203).

xabbuh committed:
[Components][ClassLoader] remove DebugClassLoader

The `DebugClassLoader` has been moved to the Debug component with
Symfony 2.4 and will be removed in 3.0 (see symfony/symfony#13203).

xabbuh commented:
It seems that we don't have docs for `ApcUniversalClassLoader`, `DebugUniversalClassLoader` and `UniversalClassLoader`.

WouterJ commented:
You forgot to remove the links from `components/class_loader/introduction.rst`

stof commented:
@xabbuh these 2 classes are deprecated. So I don't think we really need to document them

stof commented on components/class_loader/debug_class_loader.rst:
```diff
@@ -1,9 +0,0 @@
-.. index::
-    single: ClassLoader; DebugClassLoader
-    
-Debugging a Class Loader
```
The url of this page should be added in the redirection map (the only reason why it was done this way rather than removing the page was that because symfony.com was not supporting different redirections per versions of the doc, so it was impossible to remove the page)

stof commented:
I suggest removing this page in older versions

ifdattic committed:
Fix formatting

wouterj committed:
minor #4944 Fix formatting (ifdattic)

This PR was squashed before being merged into the 2.5 branch (closes #4944).

Discussion
----------

Fix formatting

| Q             | A
| ------------- | ---
| Doc fix?      | yes
| New docs?     | no
| Applies to    | 2.5
| Fixed tickets |

Commits
-------

6572fea Fix formatting

xabbuh commented:
Closing in favor of #4947.
